hashes/zkevm/src/util/expression.rs

```diff
@@ -118,6 +118,31 @@ pub mod select {
     }
 }
 
+/// Decodes a field element from its byte representation in little endian order
+pub mod from_bytes {
+    use super::{Expr, Expression, PrimeField};
+
+    pub fn expr<F: PrimeField, E: Expr<F>>(bytes: &[E]) -> Expression<F> {
+        let mut value = 0.expr();
+        let mut multiplier = F::ONE;
+        for byte in bytes.iter() {
+            value = value + byte.expr() * multiplier;
+            multiplier *= F::from(256);
+        }
+        value
+    }
+
+    pub fn value<F: PrimeField>(bytes: &[u8]) -> F {
+        let mut value = F::ZERO;
+        let mut multiplier = F::ONE;
+        for byte in bytes.iter() {
+            value += F::from(*byte as u64) * multiplier;
```

Comment:
This actually seems quite inefficient, I guess it's using F instead of Bigint.
Conversion to F is expensive because it's in Montgomery form.
If easy, chunk it by 16, convert to u128, then use F::from_u128